Source/Dafny/Compiler-Csharp.cs

```diff
@@ -99,7 +99,7 @@ public CsharpCompiler(ErrorReporter reporter)
 
     protected override BlockTargetWriter CreateStaticMain(IClassWriter cw) {
       var wr = (cw as CsharpCompiler.ClassWriter).StaticMemberWriter;
-      return wr.NewBlock("public static void Main(string[] args)");
+      return wr.NewBlock("public static void _StaticMain()");
```

Comment:
Where is this `_StaticMain` called from? I was expecting to see `_StaticMain` appear somewhere else in the PR, but I must have missed it.

Reply:
The new implementation of `EmitCallToMain` in `Compiler-Csharp` which handles halting. I'll comment a little better here.